messages/SfgeEngine.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 		"spinnerStart": "Analyzing with Salesforce Graph Engine. See %s for details."
 	},
 	"errors": {
-		"failedWithoutProjectDir": `Salesforce Graph Engine cannot run without --projectdir/-p. You must rerun your command, and either use --projectdir/-p so Graph Engine can run, or use --engine/-e to exclude Graph Engine from execution.`
+		"failedWithoutProjectDir": `Salesforce Graph Engine cannot run without --projectdir/-p. You must rerun your command, and either use --projectdir/-p so Graph Engine can run, or modify --engine/-e to exclude Graph Engine from execution.`
```

Comment:
Changed "use" to "modify", since "use" wrongly suggests they're not already using the flag, which they definitionally are.